.license

```diff
@@ -0,0 +1,4 @@
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
```

Comment:
not sure that is correct text, let's remove this file now and also this text from all php files.